docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -31,536 +33,359 @@ Clone or download the repo at [Office Add-in ASPNET SSO](https://github.com/Offi
 > [!NOTE]
 > There are two versions of the sample.
 >
-> - The **Before** folder is a starter project. The UI and other aspects of the add-in that are not directly connected to SSO or authorization are already done. Later sections of this article walk you through the process of completing it.
-> - The **Complete** version of the sample is just like the add-in that you would have if you completed the procedures of this article, except that the completed project has code comments that would be redundant with the text of this article. To use the completed version, just follow the instructions in this article, but replace "Before" with "Complete" and skip the sections **Code the client side** and **Code the server side**.
+> - The **Begin** folder is a starter project. The UI and other aspects of the add-in that are not directly connected to SSO or authorization are already done. Later sections of this article walk you through the process of completing it.
+> - The **Complete** folder contains the same sample with all coding steps from this article completed. To use the completed version, just follow the instructions in this article, but replace "Begin" with "Complete" and skip the sections **Code the client side** and **Code the server side**.
 
 Use the following values for placeholders for the subsequent app registration steps.
 
 | Placeholder           | Value                                           |
 |-----------------------|-------------------------------------------------|
 | `<add-in-name>`       | **Office-Add-in-ASPNET-SSO**                    |
-| `<redirect-platform>` | **Web**                                         |
-| `<redirect-uri>`      | `https://localhost:44355/AzureADAuth/Authorize` |
+| `<redirect-platform>` | Leave this blank. The SSO sample does not use a redirect URI. |
+| `<redirect-uri>`      | Leave this blank. The SSO sample does not use a redirect URI. |
```

Comment:
If these aren't used, why include them?

Reply:
This is an include file for Node.js and ASP.NET walkthroughs. ASP.NET does not use these. However, let me see if I can make this clearer.

Reply:
I removed these and updated the include file to indicate which value to use for Node.js vs ASP.NET.